tests/yast2_cmd/yast_dns_server.pm

```diff
@@ -48,7 +48,7 @@ sub cmd_handle {
             }
             else {
                 all { $output =~ m/\Q$_\E/i; } values %args;
-    } });
+            } });
```

Comment:
Another perltidy bug?

Reply:
No. That indentation makes more sense IMHO because the closing if/else is at the same level as the `if` now.